src/pipeline.rs

```diff
@@ -292,7 +289,7 @@ impl Pipeline {
             });
 
         render_pass.set_pipeline(&self.pipeline);
-        render_pass.set_bind_group(0, &self.uniforms);
+        render_pass.set_bind_group(0, &self.uniforms, &[]);
```

Comment:
I am not sure what the offsets here are supposed to be.

Reply:
I usually pass &[] unless you're doing something unusual